modules/scan-inactive-images.adoc

```diff
@@ -2,15 +2,37 @@
 //
 // * operating/manage-vulnerabilities.adoc
 // * operating/examine-images-for-vulnerabilities.adoc
-:_module-type: PROCEDURE
+:_content-type: PROCEDURE
 [id="scan-inactive-images_{context}"]
 = Scanning inactive images
 
-{product-title} scans all active (deployed) images every 4 hours and updates the image scan results to reflect the latest vulnerability definitions.
+{rh-rhacs-first} scans all active (deployed) images every 4 hours and updates the image scan results to reflect the latest vulnerability definitions.
 //TODO: add link to Re-scanning images.
 
-You can also configure {product-title} to scan inactive (not deployed) images automatically.
+You can also configure {product-title-short} to scan inactive (not deployed) images automatically.
 
+
+.Procedure
+
+. On the RHACS portal, navigate to *Vulnerability Management > Dashboard*.
+. On the *Dashboard* view header, select *IMAGES*.
+. Click *MANAGE WATCHES* to manage the scanning of watched images.
+. In the *MANAGE WATCHED IMAGES* dialog, enter the inactive image’s name for which you want to enable scanning.
+Verify that you enter the name of the image and not the image `id`.
+Image name is the fully-qualified image name, beginning with the registry and ending with the tag. For example, `xxxx`.
+. Select *ADD IMAGE*. RHACS then scans the image and shows the error or success message.
+. (Optional) Click *REMOVE WATCH* to remove an image from the watchlist.
++
+[IMPORTANT]
+====
+On the {product-title-short} portal, click *Platform Configuration > System Configuration* to confirm that the data retention configuration threshold is met.
+
+The image is removed from the watchlist only when the data retention configuration threshold is met.
```

Comment:
**issue:** The *Platform Configuration > System Configuration* page just lists retention days. Can you please rewrite this saying the related data (listed on the config page) form the image still appears in the RHACS UI for the number of days mentioned on that page and only removed after that period is over.

So basically, users can't do anything they should just know that image and results will be removed after retention period is over. And maybe add https://docs.openshift.com/acs/3.74/configuration/enable-alert-data-retention.html as an additional reference.